### PR TITLE
Giulio/update default theme

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd - here's the default theme cleanup!

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@caesar2164 Can you add a content attributions for the images to the `README`?

**Comment by caesar2164** [comment]
done!

File: templates/static_templates/theme-tos.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <h2 id="honor">${_('Honor Code')}</h2>
+
+    <p>${_('Put your Honor Code here!')}</p>
+
+    <h2 id="copyright">${_('Copyright')}</h2>
+
+    <p>${_('Put your Copyright Text here!')}</p>
+</section>
+
+<script>
```
Comment (stvstnfrd) [inline templates/static_templates/theme-tos.html]: I'm not really a fan of inline `script` tags; can we move this to an external file? They cache better.

File: templates/theme-footer.html [hunk 2]
```diff
@@ -1,26 +1,38 @@
...
       </nav>
     </div>
     <div class="references">
       <span>Built on <a href="http://code.edx.org">OpenEdX</a>.</span>
     </div>
+
+    <div class="copyright">
+        <p><span>
+            ${_('Copyright {year}. All rights reserved.').format(
+                year = date.today().year
```
Comment (stvstnfrd) [inline templates/theme-footer.html]: no space around `=` sign

**Comment by stvstnfrd** [comment]
eh +1 for parity.

File: templates/static_templates/theme-tos.html (hunk 1 above)
Comment (caesar2164) [inline templates/static_templates/theme-tos.html]: yeah, will do in a future PR, this brings the default theme to parity with our other themes.

**Pull request merged**